docs/IJSValueReader-api-windows.md

```diff
@@ -7,56 +7,56 @@ Kind: `interface`
 
 
 
-Forward-only reader for JSON-like streams.<br/>It is used to read data sent between native modules and the Microsoft.ReactNative.dll.<br/><br/>The JSON-like streams are data structures that satisfy the [JSON specification](https://tools.ietf.org/html/rfc8259). The data structure may have objects with name-value pairs and arrays of items. Property values or array items can be of type `Null`, `Object`, `Array`, `String`, `Boolean`, or `Number`. The `IJSValueReader` treats the `Number` type as `Int64` or `Double`. See [`JSValueType`](JSValueType).<br/><br/>When `IJSValueReader` reads data it must walk the whole tree without skipping any items. It means that if the current value type is `Object`, then we must call [`GetNextObjectProperty`](#getnextobjectproperty) to start reading its properties, and if the current type is `Array`, then we must call [`GetNextArrayItem`](#getnextarrayitem) to start reading its items.<br/><br/>See the [`IJSValueWriter`](IJSValueWriter) for the corresponding writer interface.<br/><br/>The [`IJSValueReader`](IJSValueReader) and [`IJSValueWriter`](IJSValueWriter) must be rarely used directly. Use them to create serializer and deserializer functions for a type. The rest of application code must use these functions to serialize/deserialize values. The `Microsoft.ReactNative.Cxx` and `Microsoft.ReactNative.Managed` projects offer serializer/deserializer functions for many standard types. Use them directly or to define serializer/deserializer functions for your types.
+`IJSIValueReader` is used to read data from JavaScript in custom native modules.  It acts as a stream and supports all the types in [`JSValueType`](JSValueType).

```

Comment:
Looks like the chunk of content is lost here - is this content surfaced elsewhere?

Reply:
These comment were added in PR #6891 which is not backported to the 0.64. Should we backport it before generating the docs?

---
In reply to: [589715199](https://github.com/microsoft/react-native-windows-samples/pull/400#discussion_r589715199) [](ancestors = 589715199)

Reply:
Fixed